ngonoessono_a07b.js

```diff
@@ -0,0 +1,120 @@
+/*Gertrude Vanessa Ngono Essono
+ngonoessono_a07b.js
+INFO2124
+Thoendel
+01/30/2020*/
+
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+const standardInput = process.stdin;            
+standardInput.resume();
+standardInput.setEncoding('utf8');
+const badFoods = [];
+const groceryItems = [];
+clearScreen();
+console.log(getPrompt(groceryItems));
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+
+//Step 1
+buildFoodsList (badFoods, "bread", "beer", "soda", "cheese", "chocolate");        
+
+standardInput.on('data', function (text){
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    //Step 4 Begin
+    //this block takes input from user and  match with option//
+    switch(text){
+    case "Q"||"q":
```

Comment:
you will need to declare 2 cases here instead of using or. So:
case "Q":
case "q":
    do stuff
   break;